brands/shop/toys.json

```diff
@@ -123,6 +123,15 @@
       "shop": "toys"
     }
   },
+  "shop/toys|Ri Happy Brinquedos": {
+    "countryCodes": ["pt"],
+    "tags": {
+      "brand": "Ri Happy Brinquedos",
+      "brand:wikidata": "Q10360441",
```

Comment:
Since there's a wikipedia page you should add it too: https://pt.wikipedia.org/wiki/Ri_Happy_Brinquedos

Reply:
I forgot to add, sorry